internal/cmd/cli/apply.go

```diff
@@ -210,3 +231,15 @@ func currentCommit() string {
 	}
 	return ""
 }
+
+func GetOnConflictRetries() int {
+	s := os.Getenv(FleetApplyConflictRetriesEnv)
+
+	retries := defaultApplyConflictRetries
```

Comment:
The default will only be returned when Atoi fails? 
I would prefer something like `if v, ok := os.LookupEnv("FLEET_VERSION"); ok {`, instead of relying on the Atoi` failure for empty strings. If `Atoi` fails the user likely passed a string and we could return an error to let them know.

Reply:
[os.LookupEnv](https://cs.opensource.google/go/go/+/go1.24.0:src/os/env.go;l=112) may also return "" which is a valid value for an environment variable that is set. 

We still need a valid integer, that's why it calls `Atoi`, to parse it and return an error if the value is either empty or not an `int`.

I changed the function to return the error, so we can also log it and make the user aware, and not use the default value silently.

I added that extra env variable to avoid hardcoding the number of retries and make it a bit more flexible, but I guess that it should not be a common user variable to play with.